static/js/settings_user_groups.js

```diff
@@ -340,7 +318,6 @@ export function set_up() {
         }
         const user_group = user_groups.get_user_group_from_id(group_id);
         const btn = $(this);
-
         function delete_user_group() {
```

Comment:
We should avoid such unnecessary removal (or addition also) of lines.

Reply:
@sahil839 thanks for the review. For point 2 the flow of the call event is such that

-  it changes the user group back to the saved state after each save/discard event
- previously as well it allowed editing of one user group at a time
`case "user_group":
            if (event.op === "add") {
                user_groups.add(event.group);
            } else if (event.op === "add_members") {
                user_groups.add_members(event.group_id, event.user_ids);
            } else if (event.op === "remove_members") {
                user_groups.remove_members(event.group_id, event.user_ids);
            } else if (event.op === "update") {
                user_groups.update(event);
            }
            settings_user_groups.reload();
            break;`

Reply:
@timabbott could you review this?